tests/integration/test_old_api.py

```diff
@@ -22,20 +24,18 @@ def test_old_api_create_release(responses):
 
 
 def test_old_api_upload_file(responses):
-    rf = WorkspaceFactory("workspace").create_request("request-id")
-    rf.write_file("test/file.txt", "test")
-    item = rf.get().get_path("test/file.txt")
+    request = factories.create_request("workspace", request_id="request-id")
```

Comment:
Similarly here - can we call it `release_request`, not just `request`?  This one was particularly confusing because of the removed variable called `rf` (pytest-django has a fixture `rf`, for an http `RequestFactory`)

Reply:
Good call - I'll do this as a commit in a later PR, to avoid excessive rebasing on the current stack of PRs

Reply:
never mind, will do it now!

Reply:
done